tox.ini

```diff
@@ -15,7 +15,7 @@ passenv =
     HOME
     PYTHONPATH
 deps =
-    cairo-lang==0.8.2.1
+    cairo-lang==0.9.0
```

Comment:
we should probably release + pin nile too

Reply:
Agreed